### DISCUSSION TITLE
My local Formula does not match the one I have locally

### DISCUSSION THREAD

**Discussion by scapodicasa** [Homebrew/discussions#39] [Everyday usage] [answered]
I need to install haproxy with lua support. I see the option on the github formula, but if I run `brew edit haproxy` I do not see the option so homebrew fails to install haproxy with the correct option.
Why I have a wrong formula version locally?

**Comment by Rylan12** [comment, marked as answer]
homebrew-core (the default Homebrew tap) no longer supports formulae with options so our `haproxy` formula will not contain any options. Can you link where you see the option? My guess is that it's in another tap. You'll need to run `brew tap user/tap` before you'll be able to edit their formulae.

**Reply by scapodicasa** [reply]
You are right, I was looking to an old commit.
